adoc/include/jmespath_option.adoc

```diff
@@ -1,8 +1,9 @@
 *--jq, --jmespath* 'EXPR'::
 
-Supply a JMESPath expression to apply to json output.
-Takes precedence over any specified '--format' and forces the format to be json
-processed by this expression.
+Supply a JMESPath expression to apply to JSON output.
++
+If '--format' is specified as 'TEXT', this takes precedence and forces the
+format to be JSON processed by this expression.
```

Comment:
I find the "this" in "this takes precedence" potentially confusing because the most-recently mentioned thing that it might refer to is `'TEXT'`, which is actually what loses out.

I would maybe write this as "JMESPath expressions are compatible with output `--format`s of `JSON` or `UNIX`; if `TEXT` is specified, the output will be rendered as if `JSON` was specified".

Reply:
Yes, *this* can be cleaned up. *This* is confusing.